xml/osc_building.xml

```diff
@@ -20,8 +20,120 @@
   <title>From Spec Files (RPM)</title>
-  <para/>
-  <remark>TBD</remark>
+  <para>
+   To create an RPM package you need:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     the spec file
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     a file with the extension <filename class="extension">.changes</filename>
+    </para>
+   </listitem>
+   <listitem>
+    <para>the original source archive</para>
+   </listitem>
+   <listitem>
+    <para>
+     optional patches
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     other files which does not fall into one of the previous categories
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   For existing packages, this is already the case. To build an
+   existing package, the general procedure is as follows:</para>
+  <procedure>
+   <step>
+    <para>
+     If you have not done yet, set up your project as shown in
+     <xref linkend="sec.obs.basicworkflow.setuphome"/>.
+    </para>
+   </step>
+   <step>
+    <para>
+     In the terminal, choose or create a directory on a local partition that
+     has enough space to hold the package sources.
+    </para>
+   </step>
+   <step>
+    <para>Check out the project where the package is contained:</para>
+    <screen>&prompt.user;<command>osc</command> checkout <replaceable
+     >PROJECT</replaceable> <replaceable>PACKAGE</replaceable></screen>
+    <para>
+     This creates a <filename><replaceable
+     >PROJECT</replaceable>:<replaceable>PACKAGE</replaceable></filename>
+     directory in the current directory.
+    </para>
+   </step>
+   <step>
+    <para>
+     Change the directory:
+    </para>
+    <screen>&prompt.user;<command>cd</command> <replaceable
+     >PROJECT</replaceable>:<replaceable>PACKAGE</replaceable></screen>
+   </step>
+   <step>
+    <para>
+     Decide for which build target (for example &opensuse; Tumbleweed for
+     x86_64) you want to create the RPM package and execute:
+    </para>
+    <screen>&prompt.user;<command>osc</command> build openSUSE:Tumbleweed x86_64 *.spec</screen>
+   </step>
+   <step>
+    <para>
+     Inspect the build process.
+    </para>
+    <variablelist>
+     <varlistentry>
+      <term>Successful Build</term>
+      <listitem>
+<!--       <para></para>-->
+       <screen>[   15s] RPMLINT report:
+[   15s] ===============
+[   16s] 2 packages and 0 specfiles checked; 0 errors, 0 warnings.
+[   16s]
+[   16s]
+[   16s] &wsII; finished "build <replaceable>PACKAGE</replaceable>.spec" at Fri Sep  1 11:54:31 UTC 2017.
+[   16s]
+
+/var/tmp/build-root/openSUSE_Tumbleweed-x86_64/home/abuild/rpmbuild/SRPMS/<replaceable
+ >PACKAGE</replaceable>-<replaceable>VERSION</replaceable>-0.src.rpm
+
+/var/tmp/build-root/openSUSE_Tumbleweed-x86_64/home/abuild/rpmbuild/RPMS/noarch/<replaceable>PACKAGE</replaceable>-<replaceable>VERSION</replaceable>-0.noarch.rpm</screen>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>Build ended with Error</term>
```

Comment:
title case: **E**nded
maybe also use a parallel structure and use "Unsuccessful Build" instead

Reply:
Right, I've searched for this phrase! 😆 Thanks! 👍